src/AssembleElemSolverAlgorithm.C

```diff
@@ -68,49 +69,6 @@ AssembleElemSolverAlgorithm::initialize_connectivity()
   eqSystem_->linsys_->buildElemToNodeGraph(partVec_);
 }
 
-static constexpr int simdLen = stk::simd::ndoubles;
-
-int
-calculate_shared_mem_bytes_per_thread(int lhsSize, int rhsSize, int scratchIdsSize, int nDim,
-                                      ElemDataRequests& dataNeededByKernels)
-{
-    int bytes_per_thread = (rhsSize + lhsSize)*sizeof(double) + (2*scratchIdsSize)*sizeof(int) +
-                           get_num_bytes_pre_req_data<double>(dataNeededByKernels, nDim);
-    bytes_per_thread *= 2*simdLen;
-    return bytes_per_thread;
-}
-
-template<typename T>
-void set_zero(T* values, unsigned length)
-{
-    for(unsigned i=0; i<length; ++i) {
-        values[i] = 0;
-    }
-}
-
-int get_next_num_elems_simd(int bktIndex, int bktLength)
-{
-  int numElems = simdLen;
-  if (bktLength - bktIndex*simdLen < simdLen) {
-    numElems = bktLength - bktIndex*simdLen;
-  }
-  if (numElems < 0 || numElems > simdLen) {
-    std::cout<<"ERROR, simdElems="<<numElems<<" shouldn't happen!!"<<std::endl;
-    numElems = 0;
-  }
-  return numElems;
-}
-
-size_t get_simd_bucket_length(size_t bktLength)
-{
-    size_t simdBucketLen = bktLength/simdLen;
-    const size_t remainder = bktLength%simdLen;
-    if (remainder > 0) {
-      simdBucketLen += 1;
-    }
-    return simdBucketLen;
-}
-
 struct SharedMemData {
```

Comment:
Consolidate this in `SharedMemData.h`?

Reply:
Yes. I will put that into SharedMemData.h.

Reply:
I was expecting this: 

```
In file included from /Users/sanantha/Documents/nrel/code/nalu/Nalu/src/ContinuityAdvElemKernel.C:8:
In file included from /Users/sanantha/Documents/nrel/code/nalu/Nalu/include/ContinuityAdvElemKernel.h:11:
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:94:34: warning:
      unused parameter 'lhs' [-Wunused-parameter]
    SharedMemView<DoubleType**> &lhs,
                                 ^
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:95:33: warning:
      unused parameter 'rhs' [-Wunused-parameter]
    SharedMemView<DoubleType*> &rhs,
                                ^
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:96:31: warning:
      unused parameter 'scratchViews' [-Wunused-parameter]
    ScratchViews<DoubleType> &scratchViews)
                              ^
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:103:34: warning:
      unused parameter 'lhs' [-Wunused-parameter]
    SharedMemView<DoubleType**> &lhs,
                                 ^
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:104:33: warning:
      unused parameter 'rhs' [-Wunused-parameter]
    SharedMemView<DoubleType*> &rhs,
                                ^
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:105:31: warning:
      unused parameter 'faceScratchViews' [-Wunused-parameter]
    ScratchViews<DoubleType> &faceScratchViews,
                              ^
/Users/sanantha/Documents/nrel/code/nalu/Nalu/include/Kernel.h:106:31: warning:
      unused parameter 'elemScratchViews' [-Wunused-parameter]
    ScratchViews<DoubleType> &elemScratchViews)
                              ^
235 warnings generated.
```

Reply:
@sayerhs I'm not clear whether you are saying you are seeing these warnings (meaning something needs to be changed or fixed in Kernel.h) or not?